.github/ISSUE_TEMPLATE/spec_issue.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Specification Review Issue
+about: Start a request for a specification release
+title: ''
+label: Release Review
+assignees: ''
+---
+
+## Specification issue template
+When creating a specification project lifecycle review, create issues in the  with the content defined as follows.
+- [ ] Specification name and version
+- [ ] Add the label `Release Release`
+- [ ] Naming conventions for artifacts:
+   - [ ] Specification PDF in the form of microprofile-_project_-spec-_version_.pdf
+   - [ ] Specification HTML in the form of microprofile-_project_-spec-_version_.html
+   - _project_ is the microprofile specification short project name (config, health, ...)
+   - _version_ is the two digit x.y version of the specification
+<p>
+
+- [ ] The Nexus Staging links (orgeclipsemicroprofile-NNN where NNN is the staging repository id) which contain all the binaries and relevant documentation:
```

Comment:
Maybe add the following as a comment so that people know what is expected:
`<!-- (eg, https://oss.sonatype.org/content/repositories/orgeclipsemicroprofile-1357/org/eclipse/microprofile/config/) -->`